main.cpp

```diff
@@ -1,475 +1,581 @@
 3 things it can do:
-    1)
-    2)
-    3)
- */
+    1) buy ticket to game
+    2) cheer
+    3) wear team colours
+*/
+
+struct Fan
+{
+    //name (std::string)
+    std::string name = "John Johnson";
+    //age (int)
+    int age;
+    //season ticket holder (bool)
+    bool hasSeasonTicket = true;
+    //distance from stadium (double)
+    double distanceFromStadiumInKM = 345.78;
+    //fanclub member (bool)
+    bool fanclubMember = true;
+
+    //buy ticket to game
+    void purchseTicket(float ticketPrice, std::string opponentTeam);
+    //cheer
+    void cheer(bool teamScoredGoal);
+    //wear team colours
+    void wearTeamColours(bool teamWinningStreak);
+};
 
 /*
-You've just defined 10 UDTs!
-4 of them are unrelated (UDTs 1-4).
-5 of them form the sub-objects that make up the 10th UDT. 
+Thing 8) Medic
+5 properties:
+    1) qualifications (std::string)
+    2) name (std::string)
+    3) salary (double)
+    4) explayer (bool)
+    5) IT skills (bool)
+3 things it can do:
+    1) give steriod injection
+    2) massage treatment
+    3) make fitness assessment
+*/
 
-Move them below this block comment and put them in numerical order 
-(1 - 10).
+struct Medic
+{
+    //qualifications (std::string)
+    std::string qualifications;
+    //name (std::string)
+    std::string name = "The FA Level 4 Emergency Medical Management in Football";
+    //salary (double)
+    double salary = 178909.99;
+    //explayer (bool)
+    bool isExPlayer = true;
+    //IT skills (bool)
+    bool hasITSkills = true;
+
+    //give steriod injection
+    void giveSteriodInjection(Player player, std::string bodyPart, bool servePain);
+    //massage treatment
+    void giveMassageTreatment(Player player, std::string therapyTargetArea);
+    //make fitness assessment
+    float getFitnessAssessmentScore(int restingBPM, int maxBPM, float bloodPressure); //returns value of a players fitness assessment
+};
 
-The next thing that will happen is that I will review the 
-pseudo-code that you have written.
-Send me a DM to review your pull request when the project is ready for review.
+/*
+Thing 9) Stadium
+5 properties:
+    1) number of seats (int)
+    2) pitch area (float)
+    3) number of toilets (int)
+    4) training pitch (bool)
+    5) construction cost (double)
+3 things it can do:
+    1) host home game
+    2) covid vaccination centre
+    3) require maintance
 */
 
+struct Stadium
+{
+    //number of seats (int)
+    int numSeats = 62850;
+    //pitch area (float)
+    float pitchAreaInSqMetres = 7123.89f;
+    //number of toilets (int)
+    int numToilets = 66;
+    //training pitch (bool)
+    bool trainingPitch = true;
+    //construction cost (double)
+    double constructionCost = 1200500700.99;
+
+    //host home game
+    void homeGame(std::string nameVistingTeam);
+    //covid vaccination centre
+    bool covidVaccinationCentre(int numberOfUnvaccinated, float RNumber); //returns wheteher stadium is required to be vaccination centre
```

Comment:
Your function names do not match the plain-english descriptions.
They should match almost 1:1

Your second action does not contain a verb.  You have written a noun.  
add a verb to the plain english, then fix the function name.
- [ ] revise